### ISSUE TITLE
[General] Suggest drop (again)

### ISSUE THREAD

**Issue by kittycatto1** [Catman-232/Homebrewery#15]
# Configuration
I have debated it but for the sake of user accessibility and also to avoid people from fucking up item JSONs. A configuration for `secretitem: true` items to be automatically unhidden similarly to `BypassFoodCapacity: true`, could add a little tag that makes it clear the item was a secret to avoid confusion.

# Clothes
The ability to dye the rest of the items (Capes noteably), probably already in concept, just putting it here.

# Weapons
On the behalf of Emarcee: The ability to add crown (and item?) costs to buying custom weapons from Skrit, probably with a similar configuration to remove said costs like above for accessibility sakes. Like buying the Emperor's New Fists for 5K Crowns or buying the TF2 Pan for 20 Scrap Metal as basic examples. Soft limits should be implemented to stop people from just selling items with absurd requirements like 1M crowns, this could also extend to trade items (the enchantment materials) but with even softer limits, the reason for this besides absurdity is because of how expensive people are likely gonna make enchantment costs, which become more valuable the higher level the item is.

Since we got trail colours, the ability to colour projectiles next would be great, that's about it.

# Consumables
The main thing I can suggest for Consumables is slightly more power on `consumableObject`. I wanted to suggest the idea of mixing Dye objects together to make rainbow droplets using division, for example if dyes make 30 droplets and you have 2 dye objects like blue and red, there will be 15 blue droplets and 15 red droplets.

Other than that, the ability to make some custom effects work on consumables/trade items when dropped would be great, like dealing damage to the player for dropping a fragile vase and presumedly stepping on glass shards, or making people in a radius start emoting when Dance Fever is dropped (could probably add a consent chat command that lets you choose not to be forced into emoting)

Nearly forgot as I finished up the Texture Swapper part. I wanted to suggest hot bar icons for consumables that are affected by duration (will probably need Remedy to draw for those) to make it easier to experience consumables or test items

# Texture Swapper
I've recently used `TextureReplace` to no issues but had noticed there was a few missing categories like `_font`, and was wondering if more niche containers could be added for the sake of hyperspecific customisation. I've also wanted to suggest the idea of loading `TextureReplace` folders from plugins to allow for some mods to use that instead of manual installing or config dumpings that get glued onto the profile
